ext/storage_sources/s3_store/README.md

```diff
@@ -0,0 +1,55 @@
+# S3 Store Extension
+
+## Building and running
+
+```bash
+# Create a new directory to run your build from
+$ mkdir build && cd build
+
+# Configure and run cmake
+cmake -DENABLE_PYTHON=1 -DHAVE_DIAGNOSTIC=1 -DENABLE_S3=1 -DENABLE_STRICT=0 -G Ninja ../.
+```
+
+* Running with the tag '-DENABLE_S3=1' will default to looking for a local installation of S3.
+       
+* Optionally the compiler flag 'IMPORT_S3_SDK' can be set to either:
+    -  'external' prompts the S3 to be externally installed during the build step.
+    -  'package' for the compiler to search for a local installation of the S3 to use.  
+
+* 'DENABLE_STRICT' is set to 0 to turn off strict compiler warnings so it does not pick up different formatting errors of the 3rd party
+    dependenceis.
+
+* Also optionally: '- G Ninja' here to generate a ninja build. _
+
+
+### Installing AWS SDK S3 C++ locally
+
+Follow the guide below to install SDK locally: 
+* Note: Set BUILD_ONLY flag to "s3-crt" to only install necessary dependencies for this extension. 
+    -  CMake searches locally for the s3-crt package and not s3 only package.
+
+https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/setup-linux.html
+
+
+## Testing 
```

Comment:
"Testing should be done after building WiredTiger with the S3 Extension " -- We don't need that, since the rest of the guide goes through describing that build process.